driver/docker/driver.go

```diff
@@ -52,7 +52,7 @@ func (d *Driver) Rm(ctx context.Context, force, rmVolume, rmDaemon bool) error {
 
 func (d *Driver) Client(ctx context.Context) (*client.Client, error) {
```

Comment:
Could we just add it as dial options property on `Client` instead of property of driver.

Reply:
Done and https://github.com/docker/desktop-build/pull/312 updated